person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  def initialize(id, age, parent_permission: true, name: 'unknown')
+    @id = id
```

Comment:
- [ ] Kindly consider to give a random id to this class to make your code more robust. you can use the ruby Random class. for example: `@id = Random.rand(1..1000)`